metapackages/auto-instrumentations-node/package.json

```diff
@@ -43,13 +43,13 @@
     "typescript": "4.1.3"
   },
   "dependencies": {
-    "@opentelemetry/api": "^0.18.0",
-    "@opentelemetry/instrumentation": "^0.18.0",
+    "@opentelemetry/api": "^0.18.1",
```

Comment:
should we move api to peer dependencies like in core?

Reply:
Yep but i think it make more sense in another PR